app/templates/static/form_handler.js

```diff
@@ -125,8 +125,8 @@ document.getElementById('url-form').addEventListener('submit', async function(ev
             // Initialize interact.js for drag-and-drop
             interact('.draggable').draggable({
                 modifiers: [
-                    interact.modifiers.restrict({
-                        restriction: 'parent',
+                    interact.modifiers.restrictRect({
+                        restriction: 'parent', // Restrict to the parent container
```

Comment:
I'm still able to drag it past the bottom of the image. The image is a div that includes the UI controls. That is the reason. Maybe we need an intermediate div that only covers the image and the text?

Reply:
Thank you for the feedback, @lgaleana.

I understand the issue now. The draggable text is being restricted to the parent container, which includes the UI controls. To fix this, I'll introduce an intermediate div that only covers the image and the text. This way, the draggable text will be restricted within the bounds of the image itself.

I'll make the necessary adjustments and push the changes shortly.